apps/api/v2/src/ee/bookings/2024-08-13/services/bookings.service.ts

```diff
@@ -159,9 +159,31 @@ export class BookingsService_2024_08_13 {
     );
   }
 
-  async createRegularBooking(request: Request, body: CreateBookingInput_2024_08_13) {
+  async handleBookingWithRequest(
```

Comment:
Im not sure some of these are that simple to pull from request headers. If we see in `createNextApiBookingRequest` theres a bunch more logic we do to generate the request